lib/AST/RequirementEnvironment.cpp

```diff
@@ -0,0 +1,231 @@
+//===--- RequirementEnvironment.cpp - ASTContext Implementation -----------===//
```

Comment:
Also wrong description here

Reply:
(FWIW, I fixed these in a subsequent, completely-unrelated PR)